src/ad-sizes.ts

```diff
@@ -137,6 +152,31 @@ const sizeMappings: SizeMappings = {
 			adSizes.fluid,
 		],
 	},
+	inline: {
+		mobile: [
+			adSizes.outOfPage,
+			adSizes.empty,
+			adSizes.outstreamMobile,
+			adSizes.mpu,
+			adSizes.googleCard,
```

Comment:
Answering my own question partly - `fluid` is used for our own creative templates ads e.g. interscrollers which need to take up the full width.

I should know this because I did a PR on it a few week ago :)

https://github.com/guardian/dotcom-rendering/pull/4885